tasties_app/forms.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from django.contrib.auth.models import User
+
+
+class CreateUserForm(UserCreationForm):
+    password1 = forms.CharField(max_length=16, widget=forms
+                                .PasswordInput(attrs={'class': 'form-control', 'placeholder': 'Password'}))
+    password2 = forms.CharField(max_length=16, widget=forms
```

Comment:
Let me explain:
I'm inheriting `UserCreationForm`, which already has those fields:
`password1` for the password
`password2` for password confirmation

So in those lines 7-9 I'm just overriding the existing fields because I want them to include the `form-control` class and placeholders.

I know it's not very pythonish, but in this case, I have to leave it as is.

You can see it in the [docs](https://docs.djangoproject.com/en/1.8/_modules/django/contrib/auth/forms/#UserCreationForm).

Reply:
Cool, I forgot about that. Thanks for the clarification.